quickstart.md

```diff
@@ -26,8 +26,12 @@
     Note that it typically takes only a few minutes for a new API key to have access after the API is enabled, but it can on occasion take up to an hour; until the API key is enabled, you may get errors of the form "API Key not found. Please pass a valid API key".
 
 1. **Make an AnalyzeComment request.**
-
-    Use the API key you generated in the previous step as the `key` param in this curl command:
+   
+   You can use the sample API call below to get scores directly from Perspective API models. You don’t need to build a model locally. 
+   
+   The call uses curl, which you can install on Macs or Linux with `sudo apt-get install curl`.
```

Comment:
- You can't run this apt-get command on Macs. But it should already be available on Macs, so installing it is unnecessary. Maybe clarify a bit that on Macs, it's already available.
- `apt-get` is specific to a few Linux distributions, like Debian and Ubuntu. Other Linux distributions use different commands to install packages. We should probably have more generic language about using whatever package management system is appropriate for each person's Linux distribution.